lib/index.js

```diff
@@ -127,9 +142,10 @@ EsModulesMiddlewareFactory.$inject = ['config.basePath'];
  * Connect-style middleware factory for use with express and similar servers.
  *
  * @param {string} basePath - the basePath for resolution to start from.
+ * @param {string[]} indexFiles - an array of index files to attempt to resolve directories to, in order of preference, defaults to `['index.html', 'index.js']`
  * @returns {Function} - returns the middleware which will resolve the paths based on the given basePath.
  */
-function middleware(basePath) {
+function middleware(basePath, indexFiles = ['index.html', 'index.js']) {
```

Comment:
stylistically feels a little weird that the `EsModulesMiddleWareFactory` above is broken up into multiple lines while this one is a one liner

Reply:
Thats just prettier hitting its width limit, gotta roll with that unfortunately :-)

Reply:
> what's the `examples/package-lock.json` from?
> 
> otherwise lgtm

I forgot to include it previously, the example folder needs installing separately, so we should probably include the lock file.